### PR TITLE
Update to SetupCrossgen 201 - Coherence 25722

### PR DESCRIPTION
**Pull request by mlorbetske**
This needs the `-final` packages to be available on the feed

@muratg

### CODE REVIEW

**Comment by muratg** [comment]
The packages are up on the verification feed now.

File: NuGet.Config [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
     <add key="templating" value="https://dotnet.myget.org/F/templating/api/v3/index.json" />
+    <add key="preview2-verification-feed" value="https://dotnet.myget.org/F/dotnet-2-0-0-preview2-final/api/v3/index.json" />
```
Comment (muratg) [inline NuGet.Config]: Looks good.

File: NuGet.Config [hunk 2]
```diff
@@ -4,6 +4,7 @@
...
     <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
     <add key="aspnet-final" value="https://dotnet.myget.org/F/aspnetcore-2-0-0-preview1-no-timestamp/api/v3/index.json" />
     <add key="aspnet-release" value="https://dotnet.myget.org/F/aspnetcore-release/api/v3/index.json" />
```
Comment (muratg) [inline NuGet.Config]: Why do we have all these other feeds? Doesn't seem fully right...

Comment (mikeharder) [inline NuGet.Config]: I was about to ask the same.  What is the purpose of this NuGet.config, and why does it have very old feeds?

**Comment by mlorbetske** [comment]
@dotnet-bot test this please

**Comment by livarcocc** [comment]
This needs to be rebased to release/2.0.0-preview2

**Comment by livarcocc** [comment]
@MattGertz Flow of dependencies into the CLI.

**Pull request merged**